tests/fsharp/typecheck/sigs/neg20.bsl

```diff
@@ -256,17 +264,21 @@ neg20.fs(184,34,184,39): typecheck error FS0001: This expression was expected to
 but here has type
     'obj'    
 
-neg20.fs(188,14,188,31): typecheck error FS0041: No overloads match for method 'M'. The available overloads are shown below.
-neg20.fs(188,14,188,31): typecheck error FS0041: Possible overload: 'static member C3.M : fmt:string * [<System.ParamArray>] args:string [] -> string'. Type constraint mismatch. The type 
-    'obj'    
-is not compatible with type
-    'string'    
-.
-neg20.fs(188,14,188,31): typecheck error FS0041: Possible overload: 'static member C3.M : fmt:string * [<System.ParamArray>] args:string [] -> string'. Type constraint mismatch. The type 
-    'obj'    
-is not compatible with type
-    'string []'    
-.
+neg20.fs(188,14,188,31): typecheck error FS0041: No overloads match for method 'M'.
+
+
+
+Arguments given:
+ - string
+
+ - obj
```

Comment:
I find this a little odd - Could we format as
```
Arguments given: string * obj
```
or
```
Known types of arguments: string * obj
```

Reply:
@dsyme, what formatting should be used for named arguments at call site?

Should we bother distinguishing if the method is defined with untupled arguments versus tupled?